connector_helpscout/models/helpscout_backend/common.py

```diff
@@ -73,6 +76,7 @@ class HelpscoutBackend(models.Model):
     )
 
     # These dates are used by the import crons to start where left off.
+    import_conversations_from_date = fields.Datetime()
```

Comment:
We don't need this, we're going to use web hooks to receive changes (`action_receive_hook`). We only need the ability to do a mass import once for all data, which can be handled with `binding_model.import_batch(backend)`